src/main/resources/templates/lib.ftl

```diff
@@ -471,8 +471,8 @@
   </#if>
 </#macro>
 
-<#macro transactionDate value>
-  ${formatDateTime(value, "yyyyMMdd    HHmmss")}<#t>
+<#macro transactionDate value tz>
+  ${formatDateTime(value, "yyyyMMdd    HHmmss", tz)}<#t>
```

Comment:
I think we should just use `timezone` here, like `variableLengthDateField`, instead of passing it as an argument to `transactionDate`. All transactions dates should use timezone and this way we won't forget it by mistake when adding new response templates.

Reply:
Yeah, I wasn't sure which way is the preferred way, explicitly pass it in or grab "timezone" out of thin air.  Both ways worked. Will change it.